recipes/r-workflowscriptscommon/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = '0.0.1' %}
+
+package:
+  name: r-workflowscriptscommon
+  version: {{ version }}
+
+source:
+    url: https://github.com/ebi-gene-expression-group/workflowscriptscommon/archive/v{{ version }}.tar.gz
+    sha256: abab299f7ff1c752e52d09cf5ee6260b3320fb8fcb6df3ee7a5e7413e30a5870
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+    build:
```

Comment:
This should be `host` I think. Is `git` really needed during the build-phase?

Reply:
Yep, sorry, Git was left over from earlier development. Have fixed and changed to host.